data/json/items/book/electronics.json

```diff
@@ -257,6 +257,25 @@
     "time": "45 m",
     "fun": 1
   },
+  {
+    "id": "exodii_wiring_kit_reference",
+    "type": "BOOK",
+    "category": "manuals",
+    "name": { "str_sp": "CBM interface wiring reference" },
+    "description": "A printed technical reference for the Exodii's CBM interface wire kit using numerous alien symbols along with Anglic terminology.  Thankfully someone's hand-written a legend for those symbols and terms along the margins, but it'll still take significant electronics knowledge to decipher this document.",
```

Comment:
Maybe it should just say tha Rubik themselves left some notes in the doc, since they can speak in a more understandable english if you ask  them.